decidim-accountability/spec/requests/result_search_spec.rb

```diff
@@ -59,8 +60,8 @@
     subject { response.body }
 
     it "displays all categories that have top-level results" do
-      expect(subject).to include(translated(result1.category.name))
-      expect(subject).to include(translated(result4.category.name))
+      expect(subject).to include(decidim_html_escape(translated(result1.category.name)))
+      expect(subject).to include(decidim_html_escape(translated(result4.category.name)))
```

Comment:
Missing these two
```suggestion
      expect(subject).to include(decidim_html_escape(translated(result4.category.name)))
      expect(subject).to include(decidim_html_escape(translated(result3.category.name)))
      expect(subject).to include(decidim_html_escape(translated(result4.category.name)))
```

Reply:
I am not sure i follow. in the 0.27 test, we do not have category 2 and category 3 listed as part of the test, so i did not added them in this PR.

Reply:
Ah OK, I see that the specs are different indeed between v0.27 and v0.28. Probably related to the redesign then. It's OK as is